oc_cdtapi/DmsGetverAPI.py

```diff
@@ -259,7 +259,12 @@ def _get_distr_state_info_int (self, url, parms=None):
         logging.debug ('URL = [%s]' % url)
 
         try:
-            resp = self.get (url, params=parms)
+            if 'search' in url:
```

Comment:
OK for now, but seems we may need more accuracy in the future since `search` word may be a part of another endpoint which assuming `get`.